contracts/lib/Errors.sol

```diff
@@ -699,6 +699,14 @@ library Errors {
     /// @notice Group reward pool must claim via GroupingModule.
     error IpRoyaltyVault__GroupPoolMustClaimViaGroupingModule();
 
```

Comment:
On the Errors.sol file - there are likely errors that are no longer being used and can be deleted

Reply:
I see. removed unused errors.